test/deprecated/SysBasic/chapel_specific_errors.good

```diff
@@ -1,7 +1,7 @@
+chapel_specific_errors.chpl:1: warning: The SysBasic module has been deprecated; most symbols have been moved to IO or OS as appropriate
```

Comment:
What's here is obviously no worse than what we had before, but if it were easy to quote 'SysBasic', 'IO', and 'OS', I think that would be more consistent with what's done elsewhere.  OTOH, that could be done separately (and if it affects a bunch of tests, perhaps should be).

Reply:
I believe this is actually an error message verbatim from the `deprecated "..."` piece inside the `SysBasic` module; thus, this change would not be Dyno-related. I agree that quotes would be preferable, though.